test/taxjar.spec.js

```diff
@@ -81,6 +82,16 @@ describe('TaxJar API', () => {
       assert.equal(taxjarClient.getApiConfig('apiUrl'), 'https://api.sandbox.taxjar.com/v2/');
     });
 
+    it('includes appropriate headers', () => {
+      nock(taxjarClient.getApiConfig('apiUrl'), {allowUnmocked: true}).get('/rates/12345').reply(function() {
```

Comment:
Nock's `reply` method allows passing a function to access the original request: https://github.com/nock/nock#access-original-request-and-headers